hudi-client/src/main/java/org/apache/hudi/table/CleanHelper.java

```diff
@@ -59,17 +59,17 @@
  * <p>
  * TODO: Should all cleaning be done based on {@link HoodieCommitMetadata}
  */
-public class CleanExecutor<T extends HoodieRecordPayload<T>> implements Serializable {
+public class CleanHelper<T extends HoodieRecordPayload<T>> implements Serializable {
```

Comment:
My personal feeling, `XXXHelper` usually means a utility class that contains some static helper methods. I would prefer the old one.